daprdocs/content/en/developing-applications/integrations/cloud-providers/authenticating-azure.md

```diff
@@ -0,0 +1,399 @@
+---
+type: docs
+title: "Authenticating to Azure"
+linkTitle: "Authenticating to Azure"
+description: "How to authenticate Azure components using Azure AD and Managed Identities"
+aliases:
+  - "/operations/components/setup-secret-store/supported-secret-stores/azure-keyvault-managed-identity/"
+  - "/reference/components-reference/supported-secret-stores/azure-keyvault-managed-identity/"
+---
+
+## Common Azure authentication layer
+
+Certain Azure components for Dapr offer support for the *common Azure authentication layer*, which enables applications to access data stored in Azure resources by authenticating with Azure AD. Thanks to this, administrators can leverage all the benefits of fine-tuned permissions with RBAC (Role-Based Access Control), and applications running on certain Azure services such as Azure VMs, Azure Kubernetes Service, or many Azure platform services can leverage [Managed Service Identities (MSI)](https://docs.microsoft.com/azure/active-directory/managed-identities-azure-resources/overview).
+
+Some Azure components offer alternative authentication methods, such as systems based on "master keys" or "shared keys". Whenever possible, we recommend authenticating your Dapr components using Azure AD for increased security and ease of management, as well as for the ability to leverage MSI if your app is running on supported Azure services.
+
+> Currently, only a subset of Azure components for Dapr offer support for this authentication method. Over time, we are planning to expand support to all other Azure components for Dapr. You can track the progress of the work, component-by-component, on [this issue](https://github.com/dapr/components-contrib/issues/1103).
+
+### About authentication with Azure AD
+
+Azure AD is Azure's identity and access management (IAM) solution, which is used to authenticate and authorize users and services.
+
+Azure AD is based on the OAuth 2.0 standard, which allows services (applications) to obtain access tokens to make requests to Azure services, including Azure Storage, Azure Key Vault, Cosmos DB, etc. In the Azure terminology, an application is also called a "Service Principal".
```

Comment:
Azure AD is built on top of open standards such OAuth 2.0, which allow users and services (applications)...

Reply:
Fixed